Index: cutecom-0.22.0/cutecom.desktop
===================================================================
--- cutecom-0.22.0.orig/cutecom.desktop	2010-08-12 19:00:49.704956488 +0000
+++ cutecom-0.22.0/cutecom.desktop	2010-08-12 19:01:30.321689994 +0000
@@ -1,12 +1,10 @@
 [Desktop Entry]
 Encoding=UTF-8
-BinaryPattern=
 Name=CuteCom
 MimeType=
 GenericName=Serial Terminal
 Exec=cutecom
 Icon=openterm
-TerminalOptions=
 Path=
 Type=Application
 Terminal=0
