Index: cutecom-0.22.0/CMakeLists.txt
===================================================================
--- cutecom-0.22.0.orig/CMakeLists.txt	2009-06-25 00:30:01.000000000 +0400
+++ cutecom-0.22.0/CMakeLists.txt	2009-07-13 00:20:46.029992375 +0400
@@ -40,8 +40,4 @@
 
 find_program(KDECONFIG_EXECUTABLE NAMES kde-config )
 
-if (KDECONFIG_EXECUTABLE)
-   # then ask kde-config for the kde data dirs
-   exec_program(${KDECONFIG_EXECUTABLE} ARGS --install apps --expandvars OUTPUT_VARIABLE _apps_DIR )
-   install(FILES cutecom.desktop DESTINATION ${_apps_DIR}/Utilities)
-endif (KDECONFIG_EXECUTABLE)
+install(FILES cutecom.desktop DESTINATION share/applications/kde)
